sql-ballerina/tests/client.bal

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to change the license year. It should be the year when the file first added.

Reply:
I will change it.

Reply:
I updated the year. Commit id is 56bbd50a7286fe3fa56aab2c1c581c17faa81f4c.